redbeat/schedulers.py

```diff
@@ -175,6 +179,9 @@ def __init__(self, app=None):
         self.lock_timeout = self.either_or('redbeat_lock_timeout', None)
         self.redis_url = self.either_or('redbeat_redis_url', app.conf['BROKER_URL'])
         self.redis_use_ssl = self.either_or('redbeat_redis_use_ssl', app.conf['BROKER_USE_SSL'])
+        self.redbeat_redis_cluster_options = self.either_or('redbeat_redis_cluster_options', {
+            'startup_nodes': [{"host": "localhost", "port": "30001"}]
```

Comment:
Is `localhost:30001` the right default for most people?

Reply:
In their dev, yes. 
https://redis.io/topics/cluster-tutorial This is the url followed to start with redis cluster. The default port when using their create-cluster script is 30001.
https://redis.io/topics/cluster-tutorial#creating-a-redis-cluster-using-the-create-cluster-script